### ISSUE TITLE
Add GPU to Albany Cluster

### ISSUE THREAD

**Issue by IsaiahStapleton** [nerc-project/operations#845]
I am doing some work on the Albany cluster (https://docs.google.com/document/d/1YjiJpNggYbY3fXs530VfF1AMzOKSJCYuU_hw0Ii9n-k/edit?usp=sharing) and require the use of a GPU. So I will need a GPU added to the Albany cluster.

**Comment by tzumainn** [comment]
I have no idea what the approval mechanism is for things like this, so I'll just @hpdempsey and ask for confirmation that this is okay?

**Comment by IsaiahStapleton** [comment]
Yes @tzumainn I just talked to Heidi about it and asked if she could leave an approval comment on this issue

**Comment by hpdempsey** [comment]
Yes, please let Isaiah have a GPU for the Albany cluster.
Isaiah, please tell me how long you are leasing it for.

**Comment by IsaiahStapleton** [comment]
I expect to need it until the end of January.

**Comment by tzumainn** [comment]
Okay! I've leased the node `MOC-R8PAC23U25 ` to the `albany-cluster` project until 9999-12-31; @IsaiahStapleton just let me know when you're done with it and I'll delete the lease.

**Comment by IsaiahStapleton** [comment]
@tzumainn is that lease for the end of December or end of January?

**Comment by tzumainn** [comment]
it is for the end of december in the year 9999.

**Comment by IsaiahStapleton** [comment]
Perfect thank you @tzumainn !
